csi: move csi management to admin (#17176)
rook/rook: Documentation/Getting-Started/example-configurations.md

* [`operator.yaml`](https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml): The most common settings for production deployments
    * `kubectl create -f operator.yaml`

> * [`csi-operator.yaml`](https://github.com/rook/rook/blob/master/deploy/examples/csi-operator.yaml): CRDs and custom resources for the ceph-csi-operator. The CSI operator RBAC and Deployment are included in `common.yaml` and `operator.yaml` respectively.

Reviewer: ```suggestion
* [`csi-operator.yaml`](https://github.com/rook/rook/blob/master/deploy/examples/csi-operator.yaml): Resources required for the Ceph-CSI operator, including RBAC, CRDs, and the deployment spec.
```

Author: Done
